docs/UserGuide.md

```diff
@@ -256,7 +252,7 @@ Format: `exit`
 
 ### Saving the data
 
-DorManagerPro data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually although there is an option to do so in the `export` and `import` functions.
+DorManagerPro data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually, although there is an option to do so in the `export` and `import` functions.
```

Comment:
Would you want to let import and export be hyperlinks linking to the relevant sections? Idk what would be better too